core/embed/rust/src/ui/model_mercury/flow/confirm_action.rs

```diff
@@ -118,6 +118,7 @@ fn new_confirm_action_obj(_args: &[Obj], kwargs: &Map) -> Result<Obj, error::Err
     let title: TString = kwargs.get(Qstr::MP_QSTR_title)?.try_into()?;
     let action: Option<TString> = kwargs.get(Qstr::MP_QSTR_action)?.try_into_option()?;
     let description: Option<TString> = kwargs.get(Qstr::MP_QSTR_description)?.try_into_option()?;
+    let subtitle: Option<TString> = kwargs.get(Qstr::MP_QSTR_subtitle)?.try_into_option()?;
```

Comment:
this is causing the FIDO failure
because you require the `subtitle` argument (though it is allowed to be `None`, it must be present) and `confirm_fido_reset()` is not setting it.

the call must be uhh
```suggestion
    let subtitle: Option<TString> = kwargs.get(Qstr::MP_QSTR_subtitle)?.unwrap_or(Obj::const_none()).try_into_option()?;
```